app/editor/EditorCtrl.js

```diff
@@ -190,6 +220,45 @@
       EventService.emitEvent(EventService.EVENT.GOTO_PREV_PAGE);
     };
 
+    editorCtrl.resetFilter = function() {
+      resetFilter(true);
+    };
+
+    editorCtrl.updateFilter = function() {
+      updateFilter(true);
+    };
```

Comment:
Note: I don't necessarily think that approach is better, the current way seems a bit clearer.